WordPress/WordPress-Coding-Standards
# QA: URL review across codebase

### QA: ensure URLs mentioned use https

... whenever possible.

Includes replacing some URLs with alternatives if the original URL no longer contained the relevant information, though only the URLs prefixed with `http:` have been verified, not all URLs.

Includes removing/replacing a few URLs if the page referenced either doesn't exist anymore or no longer contains the relevant information.

Including making sure that links to the PHP manual use language agnostic links.

Also, URLs used in test case files have not been verified, though some have been changed to `https`. Some have been left as `http` though to make sure that if a sniff is checking the contents of text/docs and looking for URLs, both `http` as well as `https` URLs are recognized and tested.

### QA: update/fix a number of URLs

Based on a search for [`https://make.wordpress.org/`](https://make.wordpress.org/), verified all found URLs and fixed those which needed fixing.


Closes #2064
